trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+            
+            Pmb = (Pvm + Pdm + Plm)
+            
+            p_monopole = 2*np.real(Pvm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            p_drag = 2*np.real(Pdm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            p_lift = 2*np.real(Plm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            
+            p_total = 2*np.real(Pmb*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            
+         #   print "monopole = ", p_monopole, "p_drag = ", p_drag, "p_lift = ", p_lift
+            
+          #  print "Freq = ", m*blades_number*rpm/60, "Pressure = ", pressure[m], "SPL = ", (20.0*np.log10(np.sqrt(pressure[m]**2)/(4.1784*1e-7))) #p_base)
+            
+            SPL_m_total = 20*np.log10(np.sqrt(p_total**2)/p_base)
+            SPL_m_monopole = 20*np.log10(np.sqrt(p_monopole**2)/p_base)
+            SPL_m_drag = 20*np.log10(np.sqrt(p_drag**2)/p_base)
+            SPL_m_lift = 20*np.log10(np.sqrt(p_lift**2)/p_base)         
+            
+            if SPL_m_total<0.0:
+                SPL_m_total = 0.0
+            
+            if SPL_m_drag<0.0:
+                SPL_m_drag=0.0
+            if SPL_m_lift<0.0:
+                SPL_m_lift=0.0
+            if SPL_m_monopole<0.0:
+                SPL_m_monopole=0.0
+            
+           # print "frequency = ", m*blades_number*rpm/60, "SPL_total =", SPL_m_total, "SPL_monopole =", SPL_m_monopole, \
+           #        "SPL_drag =", SPL_m_drag, "SPL_lift =", SPL_m_lift 
+            
+            total_monopole = p_monopole + total_monopole
+            total_drag = p_drag + total_drag
+            total_lift = p_lift + total_lift
+            total_pressure = p_total + total_pressure
+            
+    #************************************************************************************************************************* 
+    #***********************************************************************************************************************        
+            #SPL Calculation
+            
+            Xbig = (m*blades_number*Mach_tip*Bd)/(Mach_section*convection_factor)
+            
+            plm = np.sin(Xbig)/Xbig
+            
+            #Calculation of the Bessel function:
+            Arg    = m*blades_number*zeff*Mach_tip*np.sin(angle)/convection_factor #argument
+            NOrder = m*blades_number    #order
+            Jmb    = scipy.special.jn(NOrder, Arg)        
+            sound_speed_ft = 1115.49
+           
+            A = 538673*m*blades_number*Mach_tip*np.sin(angle)/(dist_axis*propeller_diameter*convection_factor)
+           
+            B = (np.cos(angle)/convection_factor)*thrust-(550/(zeff*zeff*Mach_tip*Mach_tip*sound_speed_ft)*hpower)
+                                                               
+            SPL[m] = 20*np.log10(A*B*plm*Jmb)
+            
+         #   SPL1 = 20*np.log10(538673*m*blades_number*Mach_tip*np.sin(angle)/(dist_axis*propeller_diameter*convection_factor) \
```

Comment:
Does this need to stay here?

Reply:
I removed this file from this PR and will put it in a noise PR